shared-helpers/src/locales/general.json

```diff
@@ -1,17 +1,25 @@
 {
   "account.accountSettings": "Account Settings",
   "account.accountSettingsSubtitle": "Account Settings, email and password",
+  "account.accountSettingsUpdate": "Update account settings",
   "account.application.confirmation": "Confirmation",
   "account.application.error": "Error",
   "account.application.noAccessError": "You are unauthorized to view this application",
   "account.application.noApplicationError": "No application with that ID exists",
   "account.application.return": "Return to applications",
+  "account.application.iconTitle": "application with checkmarks",
   "account.createAccount": "Create Account",
   "account.errorFetchingApplications": "Error fetching applications",
   "account.haveAnAccount": "Already have an account?",
   "account.myApplications": "My Applications",
+  "account.viewApplications": "View applications",
   "account.myApplicationsSubtitle": "See lottery dates and listings for properties for which you've applied",
   "account.noApplications": "It looks like you haven't applied to any listings yet.",
+  "account.signUpSaveTime.applyFaster": "Apply faster with saved application details",
```

Comment:
Do we have a follow-up ticket to get these translated?

Reply:
These have all been communicated to Sarah and captured here: https://docs.google.com/spreadsheets/d/1OfwXauzgRWfatt9LoMocbXE-ol2SvKoysKe49k0kSwQ/edit#gid=0